apps/pages/templates/pages/review_guide.html

```diff
@@ -83,7 +80,7 @@ <h2 id="tips">{{ _('Some tips for writing a great review') }}</h2>
     <ul>
       <li>
         {% trans %}
-          Submit one-word reviews such as "Great!", "wonderful," or "bad."
+          Submit one-word reviews such as "Great!", "Wonderful", or "Bad".
```

Comment:
Quotation marks are outside the comma and period for the latter two items in the list because they are part of the sentence. Quotation marks inside the comma for the first item because the exclamation point belongs to the word before it, as a unit. I'm ok with changing the example to not have an exclamation point, if it makes the grammar of the guideline easier to understand.